app/components/timeline_entry/component.html.erb

```diff
@@ -33,13 +33,29 @@
 
       <%= govuk_inset_text do %>
         <ul class="govuk-list--bullet">
-          <% description_vars[:failure_reasons].each do |failure_reason| %>
-            <li>
+          <% description_vars[:failure_reasons].each do |failure_reason| %>                
+            <% if failure_reason == description_vars[:failure_reasons].first%>
```

Comment:
This code works, but it's not very efficient, as it requires us to iterate over the failure reasons twice, and in this case we're only extracting the first item.

It might be better to structure the code where we try and pull the first one out of the list, and if it exists we render the failure reason, and then for the remaining items in the list we render them in the details component.

Something like:

```erb
<% if (failure_reason = description_vars[:failure_reasons].first).present? %>
  # failure_reason is defined within here

  <% description_vars[:failure_reasons].drop(1).each do |failure_reason| %>
    # drop first element, iterate over the rest
  <% end %>
<% end %>
```

I'd also suggest extracting the failure reason list item in to its own shared view to avoid the duplication, then we can do something like:

```erb
<%= render "shared/assessor_failure_reason_list_item", failure_reason: %>
```